include/xeus/xmessage_builder.hpp

```diff
@@ -0,0 +1,57 @@
+/***************************************************************************
+* Copyright (c) 2016, Johan Mabille, Sylvain Corlay, Martin Renou          *
+* Copyright (c) 2016, QuantStack                                           *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+#ifndef XEUS_MESSAGE_BUILDER_HPP
+#define XEUS_MESSAGE_BUILDER_HPP
+
+#include <string>
+#include <vector>
+
+#include "nlohmann/json.hpp"
+#include "xtl/xvariant.hpp"
+
+namespace nl = nlohmann;
+
+namespace xeus
+{
+    nl::json create_error_reply(const std::string& ename = std::string(),
```

Comment:
You need to prefix the method with the XEUS_API macro to avoid link errors on Windows. See [here](https://github.com/jupyter-xeus/xeus/blob/master/include/xeus/xserver_shell_main.hpp#L35) for an example. Same remark for all methods of this file.

Reply:
Only this one?